rq/scheduler.py

```diff
@@ -36,11 +37,13 @@ class RQScheduler:
 
     def __init__(self, queues, connection, interval=1, logging_level=logging.INFO,
                  date_format=DEFAULT_LOGGING_DATE_FORMAT,
-                 log_format=DEFAULT_LOGGING_FORMAT, serializer=None):
+                 log_format=DEFAULT_LOGGING_FORMAT,
+                 fallback_period=DEFAULT_SCHEDULER_FALLBACK_PERIOD, serializer=None):
```

Comment:
I don't think we need to expose this parameter as this is a strictly internal thing. After we lower this threshold to 120 seconds, I think we no longer need this parameter, keeping things simpler.

Reply:
I made a new commit removing the parameter, but keeping the value as a configuration (coming from the `DEFAULT_SCHEDULER_FALLBACK_PERIOD`). That way it should be more flexible to change in the future. Agreed that 120 would be better as well.